pkg/engine/networkinterfaces.go

```diff
@@ -305,8 +304,7 @@ func createAgentVMASNetworkInterface(cs *api.ContainerService, profile *api.Agen
 	} else {
 		dependencies = append(dependencies, "[variables('vnetID')]")
 	}
-	if !cs.Properties.OrchestratorProfile.IsPrivateCluster() &&
-		profile.LoadBalancerBackendAddressPoolIDs == nil &&
+	if profile.LoadBalancerBackendAddressPoolIDs == nil &&
```

Comment:
Add the Standard LB for node pools for private cluster + SLB + VMAS configuration